src/collar/oc_dialog.lsl

```diff
@@ -42,12 +42,14 @@ string BLANK = " ";
 integer g_iTimeOut = 300;
 integer g_iReapeat = 5;//how often the timer will go off, in seconds
 
-list g_lMenus;//13-strided list in form listenChan, dialogid, listener, starttime, recipient, prompt, list buttons, utility buttons, currentpage, button digits, auth level,extra info,  sorted
+list g_lMenus;//12-strided list in form listenChan, dialogid, listener, starttime, recipient, prompt, list buttons, utility buttons, currentpage, button digits, auth level, sorted

```

Comment:
I'm trying to think of what this extrainfo piece of data would be for.

I remember once upon a time discussing that instead of plugins having to keep track of menu UUIDs, it would be nice if they could just pass in their own arbitrary names or modes that then they would get back when the DIALOG_RESPONSE came.  That's my best guess as to why that was ever here, but yeah it doesn't look like it was being used that way.

You have any more history on this?

Reply:
Only other comment is that this looks like the kind of change where it would be really easy to miss fixing a llList2String call somewhere.  I don't have time to check for all of those right now but it seems like someone should.  Or else have some rigorous testing on it inworld.

Reply:
perhaps 'extraInfo' was supposed to be used in the future, but for a long time it has not been used at all
I got the idea to use 'extraInfo' instead of the 'g_iSelectAviMenu' and 'g_iColorMenu' global variables to select the dialog mode
do not merge yet, I will make a new pull request

Reply:
now 'g_iSelectAviMenu' and 'g_iColorMenu'  global variables are not used to change the dialog mode,
used 'extrainfo' instead

Reply:
p.s. i tested in the world it works